server/src/main/java/org/elasticsearch/action/bulk/BulkProcessor.java

```diff
@@ -398,7 +399,7 @@ public BulkProcessor add(BytesReference data, @Nullable String defaultIndex,
         try {
             ensureOpen();
             bulkRequest.add(data, defaultIndex, null, null, defaultPipeline, null,
-                true, xContentType);
+                true, xContentType, RestApiVersion.current()); //TOASKONREVIEWis this only used in server, not from Rest?
```

Comment:
can BulkProcessor be used from RestApi? I could not find usages. Should it support compatible API?

Reply:
from what we discussed on compatible-rest-api sync, BulkProcessor is not called from Rest layer, so should not need to care about the version from the RestRequest.